plugins/couchbase/src/main/java/org.radargun.service/CouchbaseOperations.java

```diff
@@ -0,0 +1,92 @@
+package org.radargun.service;
+
+import com.couchbase.client.java.document.ByteArrayDocument;
+import org.radargun.traits.BasicOperations;
+
+import java.nio.charset.StandardCharsets;
+
+public class CouchbaseOperations implements BasicOperations {
+
+   protected final CouchbaseService service;
+   private final CouchbaseCacheAdapter adapter = new CouchbaseCacheAdapter();
+
+   public CouchbaseOperations(CouchbaseService service) {
+      this.service = service;
+   }
+
+   @Override
+   public <K, V> Cache<K, V> getCache(String cacheName) {
+      return adapter;
+   }
+
+   private class CouchbaseCacheAdapter<K, V> implements BasicOperations.Cache<byte[], byte[]> {
```

Comment:
You should use something like: `private class CouchbaseCacheAdapter <K, V> implements BasicOperations.Cache<K, V>`. See as an example: https://github.com/radargun/radargun/commit/373b353317d0a05b9a7e9e7009a3c376500b8d24#diff-df93a8f6e3d2569df9283e049b55bfbc

RadarGun allows different keys and values generators. Here are some that we support https://github.com/radargun/radargun/tree/master/extensions/cache/src/main/java/org/radargun/stages/cache/generators